themes/default/content/docs/intro/pulumi-service/access-tokens.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Access Tokens"
+meta_desc: Overview of Access Tokens in the Pulumi Service for both personal accounts and organizations.
+menu:
+  intro:
+    parent: pulumi-service
+    weight: 2
+---
+## Personal Access Tokens
+
+This tab lets you manage the [access tokens](https://en.wikipedia.org/wiki/Access_token) used for logging into the Pulumi service. It provides a list of all the access tokens associated with your account, including a description and last used information for each token.
+
+When you run [`pulumi login`]({{< relref "/docs/reference/cli/pulumi_login" >}}) from the command line, you will be prompted for an access token. Once obtained, the access token logs you into the Pulumi Service and lets you interact with the Pulumi service and manage your stacks.
+
+![Access tokens](/images/docs/reference/service/access-tokens.png)
+
+### Creating Access Tokens
+
+To create an access token:
+
+1. Select your profile picture.
+1. Select **Settings**.
+1. Navigate to **Access Tokens**.
+1. Select **Create token**.
+
+### Deleting Access Tokens
+
+To delete an access token:
+
+1. Select your profile picture.
+1. Select **Settings**.
+1. Navigate to **Access Tokens**.
+1. Select the ellipsis button.
+1. Choose **Delete token**. You will be prompted in a dialog to confirm your choice.
+
+## Organization Access Tokens
+
+Organization Access Tokens provide Enterprise Growth and Business Critical customers the opportunity to manage resources and stack operations for their organization independent of a single-user account.
+
+Leveraging Organization Access Tokens for your organization's automation delivers two core benefits:
+
+* Prevents organizations from needing to pass around a Personal Access Token belonging to a privileged organization member or admin in their automated tooling. This eliminates risk with modifying or removing organization memberships, as members can be decoupled from such automation pipelines.
```

Comment:
Suggestion

Leveraging Organization Access Tokens for your organization's automation delivers many benefits over Personal Access Tokens:

* Organization Tokens belong to the organization, rather than an individual member. Any admin in the organization can view, create, and delete all Organization Tokens. If a member of your organization leaves, you don't have to worry about losing access to a core CI/CD token attached to their personal account. 
* Promotes less privileged access, as an Organization Access Token, unlike a Personal Access Token, is granted privileges only to the organization in which it was created, rather than to all organizations a single user belongs to.
* Audit logs and update history are attributed to the organization and the name of the token, rather than an individual user.

Reply:
I like the wording of these better. Included in [e3a2d04](https://github.com/pulumi/pulumi-hugo/pull/1429/commits/e3a2d04b6baacdb5515c7e330143b684314c0284)